pandas/tests/series/indexing/test_boolean.py

```diff
@@ -582,6 +582,15 @@ def test_where_dt_tz_values(tz_naive_fixture):
     assert_series_equal(exp, result)
 
 
+def test_where_list():
```

Comment:
same

Reply:
can you test where as well with array-likes for the cond

Reply:
> can you test where as well with array-likes for the cond

That's tested pretty heavily for `DataFrame` and `Series` already in fact.

Reply:
ok can u integrate this test in a related one then?

Reply:
I thought about that, but I was more comfortable leaving it as a separate test semantically speaking.  After all, it is more atomic that way IMO.

Reply:
right but it is not clear not where things are tested. we are apparently testing list-likes (ex list) somewhere else, so this is confusing to a new reader and is just future technical debt.

Reply:
Fair enough.  Integrated.